File: SDKs/ably.io/platform/1.1.0/python/src/sdk/models/shared/extras.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import push as shared_push


@dataclass_json
@dataclasses.dataclass
class Extras:
    r"""Extras
    Extras object. Currently only allows for [push](https://www.ably.io/documentation/general/push/publish#channel-broadcast-example) extra.
    """
    
    push: Optional[shared_push.Push] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('push') }, 'form': { 'field_name': 'push' }})
    

File: SDKs/ably.io/platform/1.1.0/typescript/src/sdk/models/shared/extras.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { Push } from "./push";



// Extras
/** 
 * Extras object. Currently only allows for [push](https://www.ably.io/documentation/general/push/publish#channel-broadcast-example) extra.
**/
export class Extras extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=push" })
  push?: Push;
}
